misk/src/main/kotlin/misk/digester/proto/service.proto

```diff
@@ -0,0 +1,85 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+//option java_package = "com.squareup.digester.protos.service";
+option java_outer_classname = "DigesterServiceProtos";
+
+package misk.proto.service;
+
+// Service that exposes t-digest-based histogram data.
+service DigestProvider {
```

Comment:
hmm - misk core shouldn't be exposing service interfaces. This should go into a separate module